packages/editor/src/components/editor-help/index.native.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet, PanelBody } from '@wordpress/components';
+import { useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import {
+	helpFilled,
+	plusCircleFilled,
+	alignJustifyAlt,
+	trashFilled,
+	cogAlt,
+} from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import HelpDetailNavigationScreen from './help-detail-navigation-screen.js';
+import HelpTopicRow from './help-topic-row.js';
+
+const HELP_TOPICS = [
+	{ label: __( 'What is a block?' ), icon: helpFilled },
+	{ label: __( 'Add blocks' ), icon: plusCircleFilled },
+	{ label: __( 'Move blocks' ), icon: alignJustifyAlt },
+	{ label: __( 'Remove blocks' ), icon: trashFilled },
+	{ label: __( 'Customize blocks' ), icon: cogAlt },
+];
+
+function EditorHelpTopics( { isVisible, onClose, getStylesFromColorScheme } ) {
+	const bottomSheetHeaderTitleStyle = getStylesFromColorScheme(
+		styles.bottomSheetHeaderTitle,
+		styles.bottomSheetHeaderTitleDark
+	);
+
+	return useMemo( () => (
+		<BottomSheet
+			isVisible={ isVisible }
+			onClose={ onClose }
+			hideHeader
+			hasNavigation
+		>
+			<BottomSheet.NavigationContainer animate main>
+				<BottomSheet.NavigationScreen name={ 'Topics' }>
```

Comment:
I believe we made need to internationalize this string. 
```suggestion
				<BottomSheet.NavigationScreen name={ __( 'Topics' ) }>
```